### ISSUE TITLE
Text Pads for Integrated Circuits

### ISSUE THREAD

**Issue by iamlcarus** [Aurorastation/Aurora.3#3920]
So I hooked up a simple Text Pad to a beeper circuit, the idea being that an input like "beep" would make it beep, an input like "buzz" would make it buzz. It refused to work, and at first I thought I'd done it wrong, but when I went to bay, and tried it there, it worked instantly and easily.

**Comment by Mwahahahaha** [comment]
I seemed to have no trouble with this.

Did you have a power cell inserted?
Did you configure the volume?
Did you link the text pad output pin to the beeper circuit input pin?
Did you link the text pad activator to the beeper circuit activator?
Were you inputting valid sound IDs? Bearing in mind "buzz" isn't one of those?

**Comment by lordbalkara** [comment]
**Did you have a power cell inserted?**
I used a button for debug purposes and it worked fine.
**Did you configure the volume?**
Yes
**Did you link the text pad output pin to the beeper circuit input pin?**
I'm going to assume this refers to the data pin, aka the string entered part, and if so then yes. But here's the thing, when I press enter on a string, or even specifically click the "Ok" button, it will not change the "string entered" string from null, and it will not send any data to whatever I have hooked it to.
**Were you inputting valid sound IDs? Bearing in mind "buzz" isn't one of those?**
I tried buzz on bay and found out there that it wasn't a valid input, but even beep would not work.
